test/new_tests/test_query.py

```diff
@@ -120,7 +120,7 @@ def setup_class(cls):
         except e.IndexFoundError:
             pass
 
-        if (TestBaseClass.major_ver, TestBaseClass.minor_ver) >= (7, 0):
+        if (int(TestBaseClass.major_ver), int(TestBaseClass.minor_ver)) >= (7, 0):
```

Comment:
```suggestion
        if (TestBaseClass.major_ver), TestBaseClass.minor_ver) >= (7, 0):
```
Isn't TestBaseClass.major_ver and minor_ver already initialized here?

https://github.com/aerospike/aerospike-client-python/blob/CLIENT-2383/test/new_tests/conftest.py#L174-L175

Reply:
I was experiencing errors using the 7.0 server not creating the BLOB secondary indexes, and I used that debug code to test the value.  The value for major_ver was printing 0.  I'm not sure if it was a quirky local thing or something else, so we can remove this if needed. However, I don't think it harms anything, so I would suggest leaving it.